ui/test/resource/mock-data.ts

```diff
@@ -19,6 +19,7 @@ import { ApiCollectionSummary, ICollectionSummary, ISkillSummary } from "../../s
 import { ApiReferenceListUpdate, IRichSkillUpdate, IStringListUpdate } from "../../src/app/richskill/ApiSkillUpdate"
 import { PaginatedCollections, PaginatedSkills } from "../../src/app/richskill/service/rich-skill-search.service"
 import { ApiTaskResult, ITaskResult } from "../../src/app/task/ApiTaskResult"
+import {PaginatedJobCodes} from "../../src/app/job-codes/service/job-code.service"
```

Comment:
- [ ] add space before and after { } to match other single-line imports in this file